okhttp/src/test/java/io/grpc/okhttp/OkHttpTransportTest.java

```diff
@@ -101,10 +101,4 @@ protected boolean haveTransportTracer() {
     return true;
   }
 
-  @Override
-  @org.junit.Test
-  @org.junit.Ignore
-  public void clientChecksInboundMetadataSize_trailer() {
```

Comment:
It looks like this could be made separately, since it actually has nothing to do with this particular upgrade of Netty.